config.yaml

```diff
@@ -55,6 +55,12 @@ teams:
 - name: sd-jwt-rust-maintainers
   maintainers:
   - jovfer
+- name: tsp-maintainers
+  maintainers:
```

Comment:
make yourself a maintainer, and wenjing a member. once they join, you can promote them and remove yourself